pages/bare_metal_cloud/dedicated_servers/services_backup_storage/guide.en-ie.md

```diff
@@ -341,10 +341,10 @@ This error should be resolvable by modifying the Windows Registry: Open the Wind
 
 ##### Linux
 
-Establish an SSH connection to your server, and type the following command:
+Establish an SSH connection to your server, install cifs-utils if applicable, and type the following command:
 
 ```sh
-mount -t cifs -o sec=ntlm,uid=root,gid=100,dir_mode=0700,username=root,password= //HostName/ServiceName /mnt/FolderMount
+mount -t cifs -o vers=2.0,uid=root,gid=100,dir_mode=0700,username=root,password= //HostName/ServiceName /mnt/FolderMount
```

Comment:
Hi sbraz, and thanks for comments.

About that security options, you can surely understand that update to this document came from the realization that the mount didn't work without setting vers=2.0, as 2.1 is newer, as is 3.0. I'm not sure which kind of people set vers=2.0 if it's not required.

![Screenshot 2024-12-14 at 17 08 21](https://github.com/user-attachments/assets/a1a52199-d752-42b9-a772-dd76bc8b2cd9)

About cifs-utils, is not my decision. Some people seem to try install as few packages as possible. But I see no harm unconditionally suggesting installing cifs-utils

Reply:
> came from the realization that the mount didn't work without setting vers=2.0

I actually wasn't sure why you added the version as you hadn't mentioned it anywhere in the commit message or the PR. So, even with cifs-utils installed, you had to specify a `vers=` parameter? What OS are you running?
I will perform tests on Monday.

Reply:
This is correct. vers=2.0 ( or lower ) is required for successful CIFS mount of FTP backup space.

This test was done on updated Ubuntu Noble 24.04

Reply:
Thanks, I'll reply as soon as possible after performing the tests. I want to debug this a little to see if there is any way to avoid forcing a version while mounting.